frontend/packages/db/schema/schema.sql

```diff
@@ -1337,6 +1389,12 @@ GRANT ALL ON FUNCTION "public"."set_knowledge_suggestions_organization_id"() TO
 
 
 
+GRANT ALL ON FUNCTION "public"."set_migrations_organization_id"() TO "anon";
```

Comment:
(aside) This line is added automatically, but we want not to grant anon user. We will configure `ALTER DEFAULT PRIVILEGES` in the future.